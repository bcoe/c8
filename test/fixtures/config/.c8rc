  {
    "reporter": [
      "lcov",
      "json"
    ],
    "statements": 29,
    "branches": 81,
    "lines": 78,
    "functions": 93
  }
