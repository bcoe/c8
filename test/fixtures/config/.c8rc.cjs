const config = {
  "reporter": [
    "html",
    "text"
  ],
  "lines": 32,
  "branches": "72",
  "statements": "65"
}

module.exports = config
