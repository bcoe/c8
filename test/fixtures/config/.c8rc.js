const config = {
  "reporter": [
    "html",
    "text"
  ],
  "lines": 22,
  "branches": "72",
  "statements": "65"
}

module.exports = config
