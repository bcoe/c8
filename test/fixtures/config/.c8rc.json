{
  "temp-directory": "./foo",
  "lines": 101
}
