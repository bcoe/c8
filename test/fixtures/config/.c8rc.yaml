---
reporter:
- html
- text
statements: 83
branches: 88
lines: 10
functions: 12
