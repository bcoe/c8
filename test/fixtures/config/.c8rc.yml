---
reporter:
- html
- text
statements: 89
branches: 58
lines: 69
functions: 54
