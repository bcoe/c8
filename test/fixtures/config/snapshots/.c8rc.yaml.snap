// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`parse-args --config c8 variations of config file should be able to read config files with .json, .yml, .yaml, .js, .cjs extensions should be able to resolve config file .c8rc.yaml by detection 1`] = `
",hey
i am a line of code
what
hey
what
hey
what
hey
-----------|---------|----------|---------|---------|-------------------
File       | % Stmts | % Branch | % Funcs | % Lines | Uncovered Line #s 
-----------|---------|----------|---------|---------|-------------------
All files  |   83.33 |    85.71 |      60 |   83.33 |                   
 async.js  |     100 |      100 |     100 |     100 |                   
 normal.js |      75 |    66.66 |   33.33 |      75 | 14-16,18-20       
-----------|---------|----------|---------|---------|-------------------
,"
`;

exports[`should be able to resolve config file .c8rc.yaml by detection 1`] = `
",hey
i am a line of code
what
hey
what
hey
what
hey
-----------|---------|----------|---------|---------|-------------------
File       | % Stmts | % Branch | % Funcs | % Lines | Uncovered Line #s 
-----------|---------|----------|---------|---------|-------------------
All files  |   83.33 |    85.71 |      60 |   83.33 |                   
 async.js  |     100 |      100 |     100 |     100 |                   
 normal.js |      75 |    66.66 |   33.33 |      75 | 14-16,18-20       
-----------|---------|----------|---------|---------|-------------------
,"
`;
